chainservice/builder.go

```diff
@@ -115,8 +129,11 @@ func (builder *Builder) BuildForSubChain() (*ChainService, error) {
 }
 
 // Build builds a chainservice
-func (builder *Builder) Build() (*ChainService, error) {
+func (builder *Builder) Build(opts ...BuilderOption) (*ChainService, error) {
 	builder.createInstance()
+	for _, opt := range opts {
+		opt(builder)
```

Comment:
stop height could be just an opt value, which doesn't need to be a variable of `Builder`.

Reply:
fixed